PocketCastsTests/Tests/IAP/IAPHelperTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+import StoreKitTest
+
+@testable import podcasts
+import PocketCastsServer
+
+final class IAPHelperTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    let configurationFile = "Pocket Casts Configuration"
+
+    func testRequestInfo() throws {
+        let session = try SKTestSession(configurationFileNamed: configurationFile)
+        session.clearTransactions()
+        session.resetToDefaultState()
+        session.disableDialogs = true
+
+        let helper = IAPHelper()
+        let expectation = XCTestExpectation(description: "Fetch Products")
+        NotificationCenter.default.addObserver(forName: ServerNotifications.iapProductsUpdated, object: nil, queue: nil) { notification in
+            expectation.fulfill()
+        }
+        helper.requestProductInfo()
+
+        wait(for: [expectation], timeout: 5)
```

Comment:
5s seems a lot. 😬 Is there any special reason why? Or can we reduce it to 2, maybe 1s?

Reply:
Let me test with a smaller time, when doing this type of Tests. we sometime run into delays in the CI builds.

Reply:
Will try it with 1s.